components/SpotlightPiece/SpotlightPiece.js

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+import styled from "styled-components";
+
+export default function SpotlightPiece({ pieces }) {
+  function getRandomElement(array) {
+    return array[Math.floor(Math.random() * array.length)];
+  }
+
+  const randomArtPiece = getRandomElement(pieces);
+  return (
+    <StyledSpotlightContainer>
+      <Image
+        src={randomArtPiece.imageSource}
+        height={288}
+        width={288}
+        alt={`Picture of ${randomArtPiece.name}`}
+      />
+      <h3>{randomArtPiece.name}</h3>
+      <p>{randomArtPiece.artist}</p>
+    </StyledSpotlightContainer>
```

Comment:
Here you can reuse the Artpiece component (this will be very handy later on in the project)

Reply:
I refactored with the component like 

```
<ArtPiecePreview
        image={randomArtPiece.imageSource}
        title={randomArtPiece.name}
        artist={randomArtPiece.artist}
        width={288}  
        height={288}  
      />
```
and props but I encountered performance issues: The image loading takes longer than with the current solution.